shlib/shcrypto/benchmark_test.go

```diff
@@ -41,7 +42,7 @@ func setup(b *testing.B) (*EonPublicKey, *EpochID, *EpochSecretKey) {
 		epochSecretKeyShares = append(epochSecretKeyShares, ComputeEpochSecretKeyShare(eonSecretKeyShares[i], epochID))
 	}
 	eonPublicKey := ComputeEonPublicKey(gammas)
-	assert.DeepEqual(b, new(bn256.G2).ScalarBaseMult(eonSecretKey), (*bn256.G2)(eonPublicKey), g2Comparer)
+	assert.DeepEqual(b, g2.MulScalar(new(bls12381.PointG2), g2.One(), eonSecretKey), (*bls12381.PointG2)(eonPublicKey), g2Comparer)
```

Comment:
I can see that the calculations are different here, but it could be due to the interface change. Can you confirm that it's correctly ported?